static/bidder-info/richaudience.yaml

```diff
@@ -0,0 +1,16 @@
+maintainer:
+  email: cert@richaudience.com
+gvlVendorID: 108
+modifyingVastXmlAllowed: true
+capabilities:
+  app:
```

Comment:
In bidder code it looks like you support site only. Please double check.

Reply:
in next commit i have solved:

I have change the bidder information